src/views/index.html

```diff
@@ -4,10 +4,13 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Ejs template</title>
+    <link rel="stylesheet" href="./css/styles.css" />
 </head>
 <body>
     
-    <h1>Ejs</h1>
+    <div id="titlebox">
```

Comment:
I think it's a good practice to use `class="titlebox"` instead of id=..., what do you think?
Is there any standard for this?

Reply:
I didn't know about that but I'm going to put it into practice